pytorch_lightning/utilities/xla_device_utils.py

```diff
@@ -1,104 +1,6 @@
-# Copyright The PyTorch Lightning team.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-import functools
-import importlib
-import queue as q
-import traceback
-from multiprocessing import Process, Queue
+from warnings import warn
 
-import torch
+warn("`xla_device_utils` package has been renamed to `xla_device` since v1.2 and will be removed in v1.3",
```

Comment:
deprecation and removal should be 0.2 apart, right? 1.2 and 1.4?

Reply:
fixed